docs(keyring) additional notes regarding keyring behavior (#1692)
Kong/docs.konghq.com: app/enterprise/changelog.md

* **OpenAPI2Kong**: 
    - Adds trace to errors to identify the originating YAML/JSON element that caused the error.
* **Key Authentication - Encrypted**:

>     - Provides key authentication for Routes and Services, with authentication tokens stored in an encrypted format via encryption-at-rest.

Reviewer: ```suggestion
    - Provides key authentication for Routes and Services, with authentication tokens stored in a format that is encrypted at rest.
```
